example/linalg/example_eigh.f90

```diff
@@ -0,0 +1,38 @@
+! Eigendecomposition of a real symmetric matrix 
+program example_eigh
+  use stdlib_linalg, only: eigh
+  implicit none
+
+  integer :: i
+  real, allocatable :: A(:,:),lambda(:),v(:,:)
```

Comment:
```suggestion
  real, allocatable :: A(:,:),lambda(:),vectors(:,:)
```
I suggest to use the same names as in the previous example

Reply:
[5501b83](https://github.com/fortran-lang/stdlib/pull/816/commits/5501b83f92706d99e6235821cf3aeb76de6792cb)